show-viewshed-from-point-in-scene/src/main/java/com/esri/arcgismaps/sample/showviewshedfrompointinscene/screens/MainScreen.kt

```diff
@@ -0,0 +1,287 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.showviewshedfrompointinscene.screens
+
+import android.app.Application
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import com.esri.arcgismaps.sample.showviewshedfrompointinscene.components.ComposeSceneView
+import com.esri.arcgismaps.sample.showviewshedfrompointinscene.components.SceneViewModel
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String, application: Application) {
+    // create a ViewModel to handle SceneView interactions
+    val sceneViewModel = SceneViewModel(application)
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Box {
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(it)
+                ) {
+                    // composable function that wraps the SceneView
+                    ComposeSceneView(
+                        modifier = Modifier.fillMaxSize().weight(1f),
+                        sceneViewModel = sceneViewModel
+                    )
+                    // sliders
+                    HeadingSlider(sceneViewModel)
+                    PitchSlider(sceneViewModel)
+                    HorizontalAngleSlider(sceneViewModel)
+                    VerticalAngleSlider(sceneViewModel)
+                    MinimumDistanceSlider(sceneViewModel)
+                    MaximumDistanceSlider(sceneViewModel)
+                    // checkbox
+                    Row {
+                        FrustumCheckBox(sceneViewModel)
+                        AnalysisCheckBox(sceneViewModel)
+                    }
+                }
+
+            }
+        }
+    )
+}
+
+@Composable
+private fun HeadingSlider(sceneViewModel: SceneViewModel) {
+
+    var sliderValue by remember {
+        mutableStateOf(82f)
+    }
+    Row {
+        Text(modifier = Modifier.padding(start = 10.dp, top = 10.dp, end = 10.dp), text = "Heading")
+        Slider(
+            modifier = Modifier.weight(1f),
```

Comment:
All the sliders have different width sizes, and the slider's width changes when the text value is more than 2 digits which seems to be Compose funky behavior. 

Could you update the text value to be a set defined size, and all sliders to follow the same size width? Similar to the iOS sample:
![image](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/c08b35f2-1846-465c-a856-35eaf3061633)